tasks/synopsys-detect-task/tests/DetectUtilsTest.ts

```diff
@@ -0,0 +1,20 @@
+import {parseArguments} from "../ts/DetectUtils";
+
+const assert = require('assert')
+
+describe('DetectUtils tests', function () {
+
+    it('validate argument parsing', function() {
+        const firstKey = "first.key"
+        const firstValue = "test"
+        const secondKey = "second.key"
+        const secondValue = "another"
+
+        const detectArgs: string = `--${firstKey}=${firstValue} \n--${secondKey}=${secondValue}`
```

Comment:
It might be relevant that the newline string on windows is not '\n', but '\r\n'.